build.gradle

```
@@ -3,7 +3,7 @@ plugins {
     id 'eclipse'
     id 'idea'
     id 'maven-publish'
-    id 'net.neoforged.gradle.userdev' version '7.0.142'
+    id 'net.neoforged.moddev' version '1.0.0'
```

Maybe use the latest stable version?

```suggestion
    id 'net.neoforged.moddev' version '1.0.19'
```